public/js/viewer.js

```diff
@@ -94,17 +115,25 @@ var View = (function () {
     malate_2: new Node(new THREE.Vector3(-2, -6, 0), "malate", true),
   };
 
+  var placeholderNodes = {
+    dihydroxyacetone_phosphate_glyceraldehyde_3_phosphate_placeholder: new Node(midPointVector([
+      existingNodes.dihydroxyacetone_phosphate,
+      existingNodes.glyceraldehyde_3_phosphate,
+    ]), "placeholder", false, true),
+  };
 
+  var nodes = {
+    ...existingNodes,
```

Comment:
do we need to support IE11? spread syntax won't work in this case

Reply:
We don't need to support IE11, thank you for checking and letting me know. I will make a note in case this is a problem in the future